paimon-core/src/main/java/org/apache/paimon/catalog/Catalog.java

```diff
@@ -108,6 +111,31 @@ void dropDatabase(String name, boolean ignoreIfNotExists, boolean cascade)
      */
     List<String> listTables(String databaseName) throws DatabaseNotExistException;
 
+    /**
+     * Get names of all tables under this catalog. An empty list is returned if none exists.
+     *
+     * <p>NOTE: System tables will not be listed.
+     *
+     * @return a list of the names of all tables in this catalog
+     */
+    default List<Identifier> listTables() {
```

Comment:
This interface appears to be confused.
List tables from current database or list all tables?

Reply:
> This interface appears to be confused. List tables from current database or list all tables?

I changed it to `listAllTables`, Is this okay?  I want to get all the tables under the catalog.